requirements.in

```diff
@@ -1,4 +1,4 @@
-https://github.com/codecov/shared/archive/f8e95d52d6182d4fc4d5492d5eea50e045a8b51c.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/a0b5d256b6d580a99dd3a1e900f5c9bdaf2d5c9e.tar.gz#egg=shared
```

Comment:
Our latest shared is after this one, can you update to make sure you don't accidentally go back?

Reply:
Done.